src/docs/product/releases/setup/index.mdx

```diff
@@ -1,57 +1,85 @@
 
 </Note>
 
-Notifying Sentry of a release enables auto discovery of which commits to associate with a release and identifies what we consider "the most recent release" when searching in sentry.io. In addition, telling Sentry of a release helps identify new issues, regressions, and whether an issue is resolved in the next release. You do not need a repository integration for any of these features, though we recommend [installing a repository integration](/product/releases/setup/release-automation/) for efficiency.
+### Release Health
 
-The behavior of a few features depends on whether a project is using semantic or time-based versioning.
+Release health is supported for the following SDKs:
 
-- Regression detection
-- `release:latest`
+- [Android](/platforms/android/configuration/releases/#release-health)
+- [Flutter](/platforms/flutter/configuration/releases/#release-health)
+- [iOS](/platforms/apple/configuration/releases/#release-health)
+- [JavaScript (browser)](/platforms/javascript/configuration/releases/#release-health)
+- [JavaScript (node)](/platforms/node/configuration/releases/#release-health)```
+- [Native (C/C++)](/platforms/native/configuration/releases/#release-health)
+- [Python](/platforms/python/configuration/releases/#release-health)
+- [React Native](/platforms/react-native/configuration/releases/#release-health)
+- [Rust](/platforms/rust/configuration/releases/#release-health)
+- [.NET](/platforms/dotnet/configuration/releases/#release-health)
+- [Unity](/platforms/unity/configuration/releases/#release-health)
 
-We automatically detect whether a project is using semantic or time-based versioning based on:
+## 2. Upload Source Maps or Other Debug Files
 
-- If ≤ 2 releases total: we look at most recent release.
-- If 3-10 releases total: if any of the most recent 3 releases is semver, project is semver.
-- If 10+ releases: if any of the most recent 3 releases is semver, and 3 out of the most recent 10 releases is semver, then the project is semver.
 
-### Using Release Automation
+<Note> 
 
-If you're using [Release Automation](/product/releases/setup/release-automation/), let Sentry know you’ve deployed by sending an additional request after creating a release:
+Releases must be set up before source maps, so if you haven't yet done that, go back to [step 1](#configure-your-sdk).
+
+</Note>
+
+In cases where your code is processed - bundled, minified, transpiled, compiled, or otherwise obfuscated by tools like Webpack, Terser, or the C++ compiler -  source maps (or other debug files) are necessary for Sentry to display the source code in your stack traces in its original untransformed form. Setting up source maps is also necessary before [suspect commits](#3-enable-suspect-commits) can be enabled. Check out our SDK documentation to find source map or other debug file set up instructions for your specific platform if it's not in the list below:
```

Comment:
```suggestion
In cases where your code is processed — bundled, minified, transpiled, compiled, or otherwise obfuscated by tools like Webpack, Terser, or the C++ compiler —  source maps (or other debug files) are necessary for Sentry to display the source code in your stack traces in its original untransformed form. Setting up source maps is also necessary before [suspect commits](#3-enable-suspect-commits) can be enabled. Check out our SDK documentation to find source map or other debug file set up instructions for your specific platform if it's not in the list below:
```